### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Certificates_Screen Reader: Narrator is not reading the headings of the page

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#63]
**Environment Details:**
URL: https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
MAS ID Violated: MAS1.3.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Certificates and press enter
6) Start Screen Reader (Narrator/NVDA).
7) Verify screen reader is reading all the contents, controls, links, images properly throughout the    process.

**Actual Result:**
Narrator is not reading the heading "Certificates" when the user opens the page from the left pane list items.

**Expected Result:**
When the user open any of the list items present in the left pane, narrator should read the headings of the page properly. 

**User Impact:** 
User would not be able to understand in which page currently they are if Screen Reader does not read the headings of the page properly. 
Here in this page Narrator should read the heading as "Web Servers- Certificates" page which will be helpful for the user to understand the current page

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by Shayanti123** [comment]
Attaching video for the issue:
[Bug_63_Narrator is not reading the heading of the page.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317444/Bug_63_Narrator.is.not.reading.the.heading.of.the.page.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This issue is not applicable because UI  is changed. 
I confirmed that users can navigate all the controls with using down-arrow key.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Narrator; #Edge

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
